### DISCUSSION TITLE
脱离DMM启动相关问题

### DISCUSSION THREAD

**Discussion by q9q9qq92007** [UmamusumeResponseAnalyzer/UmamusumeResponseAnalyzer#13] [Q&A] [answered]
作者你好，请问说明文件中提到的DMM Game Player β指的是什么？
另外个人有一台笔记本还在使用win7系统，运行程序后发现由于不支持ANSI转义代码而报错，想请教一下有比较简单的解决方案吗？

**Comment by EtherealAO** [comment, marked as answer]
就是现在用的
![image](https://user-images.githubusercontent.com/28262778/166824470-c3be18fb-2195-453a-bc3b-82f97730c04c.png)
这个
win7由于缺少测试机器，马娘在虚拟机里无法正常运行，但是似乎只有prompt不能在WIN7下正常显示，你可以试试 https://github.com/EtherealAO/UmamusumeResponseAnalyzer/releases/tag/win7-test 这个版本能不能正常用其他功能

**Reply by q9q9qq92007** [reply]
非常感谢！更新和启动都正常了，不过进入游戏后还是抓不到数据，怀疑可能是没有成功发包的问题
测试后发现本地打开马娘时EXNOA-CarrotJuicer的控制台能正常启动（但是马娘无法通过直连启动），而通过DMMGAMEPLAYER则会卡在加载DLL的过程中，应该是没辙了

**Reply by EtherealAO** [reply]
若非使用修改版的[umamusume-localify](https://github.com/EtherealAO/umamusume-localify)，则需要在对应的插件配置中将URL设置为http://127.0.0.1:4693

notifier_host必须为http://127.0.0.1:4693 ，不能是 http://127.0.0.1:4693/ 。

**Reply by q9q9qq92007** [reply]
已经换成了整合版本的umamusume-localify，但通过DMMGAMEPLAYER打开马娘时仍然有类似的问题（使用Win8/Win10或本地路径启动都可以正常运行，可能是Win7和DMM独有的BUG）
通过控制台检测时，会发现卡在如下的界面
![QQ图片20220510165416](https://user-images.githubusercontent.com/22311112/167590740-bcf0e4cc-3139-43b7-bbef-aefef0cb5610.png)
之后无论在游戏中进行什么操作都没有响应了

**Reply by EtherealAO** [reply]
win7已经不受正式支持了
